physics/SFC_Layer/UFS/sfc_nst.f90

```diff
@@ -26,8 +26,8 @@ module sfc_nst
   !> \section NSST_general_algorithm GFS Near-Surface Sea Temperature Scheme General Algorithm
   subroutine sfc_nst_run                                          &
        ( im, hvap, cp, hfus, jcal, eps, epsm1, rvrdm1, rd, rhw0,  &  ! --- inputs:
-       pi, tgice, sbc, ps, u1, v1, t1, q1, tref, cm, ch,          &
-       lseaspray, fm, fm10,                                       &
+       pi, tgice, sbc, ps, u1, v1, usfco, vsfco, icplocn2atm, t1, &
+       q1, tref, cm, ch, lseaspray, fm, fm10,                                       &
```

Comment:
@binli2337 please align the "&" properly with neighbor lines. Thanks!

Reply:
Done